data/address-formats.json

```diff
@@ -31,55 +31,55 @@
                 "pt", "se", "si", "sk", "sm", "va"
             ],
             "format": [
-                ["street", "housenumber"],
+                ["unit","street", "housenumber"],
                 ["postcode", "city"]
             ]
         },
         {
             "countryCodes": ["fr", "lu", "mo"],
             "format": [
-                ["housenumber", "street"],
+                ["unit","housenumber", "street"],
                 ["postcode", "city"]
             ]
         },
         {
             "countryCodes": ["br"],
             "format": [
-                ["street"],
+                ["street","unit"],
                 ["housenumber", "suburb"],
                 ["city", "postcode"]
             ]
         },
         {
             "countryCodes": ["vn"],
             "format": [
-                ["housenumber", "street"],
+                ["housenumber", "street","unit"],
```

Comment:
I don’t think we should include `addr:unit` in the [Vietnam address format. There’s only [one (mistaken) use](http://overpass-turbo.eu/s/r2X) of this tag in all of Vietnam, reflecting the fact that units aren’t numbered in that country. Floor numbers are sometimes given in addresses, but generally the smallest level of addressing is the building number.

Addresses can be quite granular even without `addr:unit`, because the `addr:housenumber` and `addr:street` tags are both hierarchical. For example, this is the address of [building 8, alley 251, lane 48, Nguyễn Khang street](http://www.openstreetmap.org/way/351348266):

> Số 48/251/8 đường Nguyễn Khang
> Xã Yên Hòa
> Quận Cầu Giấy
> Hà Nội

```
addr:housenumber=48/251/8
addr:street=Nguyễn Khang
addr:subdistrict=Yên Hòa
addr:district=Cầu Giấy
addr:city=Hà Nội
```

`addr:housenumber` can be even more complex, for example [`7/14/12/3/23`](http://www.openstreetmap.org/node/5021351015). On the other hand, some mappers prefer to put all the alley and lane numbers in `addr:street` so the map doesn’t look so cluttered, for example [lane 19, Liễu Giai street](http://www.openstreetmap.org/node/4453205990).

More information about Vietnam’s address formats:

https://vi.wikipedia.org/wiki/Địa_chỉ
https://youbianku.com/files/upu/VNM.pdf
http://vforum.vn/diendan/showthread.php?91066-Thon-Xom-Ap-Xa-Phuong-Huyen-Quan-Tinh-Thanh-Pho-trong-tieng-Anh-la-gi-khi-viet-dia-chi-nha

/ref #2252